funcao-divisao/divisao.js

```diff
@@ -0,0 +1,12 @@
+function divisao(x, y) {
+  const z = x / y;
```

Comment:
E se o y for negativo?
Volta lá no card, pois precisa adicionar aqui uma verificação.

Reply:
feito!